test/firrtl/lower-to-rtl.mlir

```diff
@@ -39,5 +39,17 @@
 
     // CHECK-NEXT: firrtl.connect %out1, %8 : !firrtl.flip<uint<4>>, i4
     firrtl.connect %out1, %5 : !firrtl.flip<uint<4>>, !firrtl.uint<4>
+
+    // CHECK-NEXT: %10 = rtl.wire {name = "test-name"} : i4
+    firrtl.wire {name = "test-name"} : !firrtl.uint<4>
+
+    // CHECK-NEXT: %11 = rtl.wire : i2
+    firrtl.wire : !firrtl.uint<2>
```

Comment:
Great testcases!  Please propagate the name attribute as well, rtl.wire supports an optional name attribute.

Reply:
Thank you for all the feedback. I'm a little confused by this comment, though, because I'm interpreting "propagate the name attribute" as "make sure the RTL wire has the same name as the FIRRTL wire, if it was specified," and the test with the "test-name" name on the FIRRTL wire is checking for this.

Reply:
Oh, I completely misread this - you're right, this is already covered - thanks again and please disregard this comment :)